packages/react-vapor/docs/src/OneDemoToRuleThemAll.tsx

```diff
@@ -21,7 +21,7 @@ function TopNav() {
 function Header() {
     return (
         <div id="header" className="header flex flex-center space-between p2" style={{minHeight: '65px'}}>
-            <h1 className="h1">
+            <h1 className="h1 text-white">

```

Comment:
Why is this `text-white` required? Should the default styling be correct for this already? :P